admin-ui/src/pages/request-replay-page.tsx

```
@@ -0,0 +1,438 @@
+
+  const handleSend = useCallback(async () => {
+    if (!form || !requestId) return
+
+    const overrides = buildOverrides()
+
+    if (form.mode === "collect") {
+      setSending(true)
+      setCollectResult(null)
+      setLiveEvents(null)
+      try {
+        const res = await replayCollect(requestId, { accountId: form.accountId, overrides })
```

ESLint

[error] Replace `·accountId:·form.accountId,·overrides` with `⏎··········accountId:·form.accountId,⏎··········overrides,⏎·······`

(prettier/prettier)